docs/bouwblokken/publieke-inkoop/index.md

```diff
@@ -1,77 +1,141 @@
 
-Bij het publiek inkopen van software met bijbehorende algoritmen en AI zijn de wensen van de behoeftesteller en de doelstellingen van de organisatie van groot belang. Dit kan tot verschillende situaties leiden:
+- inkoop van een kant-en-klare oplossing
+- aanpassen van een bestaande oplossing
+- ontwikkelen van een nieuwe oplossing
 
-•	Een al ontwikkelde kant-en-klare oplossing voldoet direct aan deze wensen en doelstellingen;
+De eisen voor **kant-en-klare oplossingen** vind je in de laatste 2 levenscyclus-fases: [controle en onderhoud](https://minbzk.github.io/Algoritmekader/levenscyclus/monitoring-en-beheer/) en [stoppen](https://minbzk.github.io/Algoritmekader/levenscyclus/uitfaseren/).
 
-•	Een al ontwikkelde oplossing moet eerst worden aangepast voordat deze kan worden gebruikt;
+De eisen voor **oplossingen aanpassen of ontwikkelen** vind je in de laatste 7 levenscyclus-fasen: [ontwerpen](https://minbzk.github.io/Algoritmekader/levenscyclus/ontwerp/), [data onderzoeken en voorbereiden](https://minbzk.github.io/Algoritmekader/levenscyclus/dataverkenning-en-datapreparatie/), [ontwikkelen](https://minbzk.github.io/Algoritmekader/levenscyclus/ontwikkelen/), [beoordelen](https://minbzk.github.io/Algoritmekader/levenscyclus/verificatie-en-validatie/), [implementeren](https://minbzk.github.io/Algoritmekader/levenscyclus/implementatie/), [controle en onderhoud](https://minbzk.github.io/Algoritmekader/levenscyclus/monitoring-en-beheer/) en [stoppen](https://minbzk.github.io/Algoritmekader/levenscyclus/uitfaseren/). 
 
-•	Er moet een nieuwe oplossing worden ontwikkeld om te voldoen aan de wensen. 
+## Voldoen aan de regels
 
-Deze inschatting is dus bepalend wat wel en niet van een product mag worden verwacht. Dit is relevant voor zowel de leverancier als de opdrachtgever. Het is aannemelijk dat als het om risicovolle (nog te ontwikkelen) algoritmen of AI gaat, de opdrachtgever een intensieve bijdrage moet leveren aan de samenwerking om het product te kunnen gebruiken. De opdrachtgever zal bijvoorbeeld moeten aangeven wat de juridische en ethische grenzen zijn van de uiteindelijk werking van het algoritme of AI. Als een kant-en-klare oplossing wordt afgenomen, dan zal de leverancier moeten laten zien dat de ontwikkelde algoritmen en AI voldoen aan alle vereisten en moet dit kunnen aantonen. 
+<details>
+<summary>Eisen publieke inkoop</summary>
 
-De inzichten uit de algoritme levenscyclus kunnen ondersteunen bij bijvoorbeeld de behoeftestelling, het maken van make-or-buy beslissingen, de te hanteren aanbestedingsvorm, de totstandkoming van de selectie- en gunningseisen, contractspecificaties en de uitvoering en management van het contract. De algoritme levenscyclus kan worden geraadpleegd via het tabblad boven aan deze pagina. Per fase en per type algoritme of AI kan worden bekeken aan welke vereisten moet worden voldaan en welke beheersmaatregelen kunnen worden getroffen. 
 
-## Vereisten
```

Comment:
Waarom is dit kopje met deze informatie weggehaald? Dit is een afspraak die we hebben gemaakt wat betreft structuur. Suggestie/voorstel?